src/domManipulation.ts

```diff
@@ -4,16 +4,27 @@ import fetchWeatherForecast from "./api";
 
 // Get the parent element for the main cities/locations which holds all the buttons.
 const mainCities = document.getElementById("main-cities-nav");
```

Comment:
Should include the element type:
`document.getElementById<HTMLDivElement>("main-cities-nav");`